commit f5842466af736ab7efffdf97e7134355e71f866c (refs/changes/68/1268/2)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2017-01-17T23:10:37+00:00 (2 years, 9 months ago)
    
    MANTA-3095 mako test suite never reads its HTTP streams
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/test/api.test.js b/test/api.test.js
index 5e86f06..acbf24d 100644
--- a/test/api.test.js
+++ b/test/api.test.js
@@ -54,6 +54,7 @@ var getNonexistentObject = function (t) {
         var req = http.request(options, function (res) {
                 console.log('STATUS: ' + res.statusCode);
                 console.log('HEADERS: ' + JSON.stringify(res.headers));
+                res.resume();
                 t.equal(res.statusCode, 404);
                 t.end();
         });
@@ -84,6 +85,7 @@ test('put 10 MiB object', function (t) {
         var req = http.request(options, function (res) {
                 console.log('STATUS: ' + res.statusCode);
                 console.log('HEADERS: ' + JSON.stringify(res.headers));
+                res.resume();
                 t.equal(res.statusCode, 201);
                 t.end();
         });
@@ -155,6 +157,7 @@ test('100s of small files', function (t) {
 
                         var req = http.request(options,
                             function (res) {
+                                res.resume();
                                 t.equal(res.statusCode, 201);
                                 return (subcb(null));
                         });
